config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/OutboundJmsDestinationDetail.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+import static java.util.Arrays.stream;
+
+public class OutboundJmsDestinationDetail extends JmsDestinationDetail {
+    
+    private boolean isTemplate;
+    private MessageFormat messageFormat;
+    private PoolingType poolingType;
+    private SessionPoolingSettings sessionPoolingSettings;
+    private ConnectionPoolingSettings connectionPoolingSettings;
+
+    public OutboundJmsDestinationDetail() {
+    }
+    
+    public OutboundJmsDestinationDetail(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 8 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=138&issues=AWdc315sy5Us1olWmrcg&open=AWdc315sy5Us1olWmrcg)

[ISSUE_KEY]: AWdc315sy5Us1olWmrcg